services/web/server/src/simcore_service_webserver/projects/projects_folder_handlers.py

```diff
@@ -0,0 +1,75 @@
+""" Handlers for CRUD operations on /projects-with-folder/{*}
+
+"""
+
+import logging
+
+from aiohttp import web
+
+from .._meta import api_version_prefix as VTAG
+from ..login.decorators import RQT_USERID_KEY, login_required
+from ..security_decorators import permission_required
+from .projects_folder_db import APP_PROJECT_FOLDER_DBAPI, ProjectFolderDB
+from .projects_handlers import get_project, list_projects
+
+log = logging.getLogger(__name__)
+
+
+routes = web.RouteTableDef()
+
+
+@routes.get(f"/{VTAG}/projects-with-folder")
+@login_required
+@permission_required("project.read")
+async def list_projects_with_folder(request: web.Request):
+    db: ProjectFolderDB = request.config_dict[APP_PROJECT_FOLDER_DBAPI]
+    projects = await list_projects(request)
+    await db.add_folder_to_projects(projects["data"])
+    print("projects1", projects)
```

Comment:
use the log, not print.
and I am not sure you want to print all the projects returned here... or at least only as a debug message.

Reply:
clean up missing.... removed